UIforETW/ChildProcess.cpp

```diff
@@ -214,14 +219,14 @@ void ChildProcess::WaitForCompletion(bool printOutput)
 	if (hChildThread_)
 	{
 		WaitForSingleObject(hChildThread_, INFINITE);
-		CloseHandle(hChildThread_);
+		handle_close::closeHandle(hChildThread_);
 		hChildThread_ = 0;
 	}
 
 	// Clean up.
 	if (hPipe_ != INVALID_HANDLE_VALUE)
 	{
-		CloseHandle(hPipe_);
+		handle_close::closeHandle(hPipe_);

```

Comment:
static analysis picks up misuse - code like:

```
handle_close::closeHandle(someHandle);
```
(several hundred lines of code in between)
```
handle_close::closeHandle(someHandle);
```
triggers a code analysis warning/error